### PR TITLE
apply coding in artik_demo.c

### PR DESCRIPTION
**Pull request by ruri980**
enter error

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [17ee7ad960db5f4323d1a32c0077e188a539c5e8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [17ee7ad960db5f4323d1a32c0077e188a539c5e8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1673ae378002b6f4a5b8f80032c67dd30a8134c7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [1673ae378002b6f4a5b8f80032c67dd30a8134c7] - Code Rule Check Result:
apps/examples/aws/aws_sample_main.c:102: ERROR: [IDT_M_SCH] switch and case should be at the same indent

File: apps/examples/aws/aws_sample_main.c [hunk 1]
```diff
@@ -100,24 +100,24 @@ int aws_sample_cb(void *args)
...
-		case 4:
-			aws_subscribe_publish_library(0, NULL);
-			break;
-		case 5:
-			aws_subscribe_publish(0, NULL);
-			break;
-		default:
-			printf("Wrong number : %d\n", num);
-			break;
+			case 1:
```
Comment (sunghan-chang) [inline apps/examples/aws/aws_sample_main.c]: I am sorry but previous one is right.
`case`  should have a same indendation with `switch`.

**Comment by sunghan-chang** [comment]
@ruri980 Additionally we need to squash two commits as a commit. Because it is a same kind of change.
```
git rebase -i HEAD~2
pick 17ee7ad apply coding in artik_demo.c
pick 1673ae3 Apply Coding Style   (HERE!! Let's change 'pick' to 'fixup')
ctrl+x
git push -f
`

**Comment by sunghan-chang** [comment]
@ruri980 This fix is not right. Let's close now, when you are free, please fix again and reopen this. Thank you for contribution.

**Pull request closed without merging**